README.md

```diff
@@ -6,11 +6,18 @@
 
 ## Setup
 
-1) We're going to use the `activerecord` gem to create a mapping between our
-database and model.
-
-2) While this is marked as a Readme, fork and clone this repository. Take a
-look at the `Gemfile` in this directory. Be sure to run `bundle install`.
+1. We're going to use the `activerecord` gem to create a mapping between our
+   database and model.
+2. While this is marked as a Readme, you should code-along with this document.
+   If you have access to a Flatiron School sandbox (you'll see a blue button up
+   top), launch it. If you're working on a local system, simply open a new
+   terminal window.
+3. Once you're in a terminal environment, issue `git clone <clone source
+   path>`. You'll find the path by clicking on the "Clone or download" button
+   at the [repo][].
+4. `cd` into the cloned directory e.g. `cd mechanics-of-migrations-v-000/`
+5. Ensure you can `bundle` in the directory: `bundle install`
+6. If that works, then you're ready to proceed.
```

Comment:
I can imagine students saying, "And what if it didn't work?!?!"

Reply:
I assume that's what AAQ is for....

Reply:
Amended, hopefully in a generic way.

Reply:
Yeah haha, you're def right about that — most of that burden falls upon AAQ. Guess it really comes down to more of a product call on how that burden should be split between excruciatingly exhaustive curriculum (more effort to produce / update) and greater reliance on AAQ (which would eventually probably necessitate more staffing)